docs/en/ingest-management/fleet/fleet-settings-output-logstash.asciidoc

```diff
@@ -99,3 +103,75 @@ output is set in the <<agent-policy,agent policy>>.
 | When this setting is on, {agent}s use this output to send <<monitor-elastic-agent,agent monitoring data>> if no other output is set in the <<agent-policy,agent policy>>.
 
 |===
+
+
+[[ls-output-settings-yaml-config]]
+== Advanced YAML configuration
+
+[cols="2*<a"]
+|===
+| Setting | Description
+
+include::../elastic-agent/configuration/outputs/output-logstash.asciidoc[tag=backoff.init-setting]
+
+// =============================================================================
+
+include::../elastic-agent/configuration/outputs/output-logstash.asciidoc[tag=backoff.max-setting]
+
+// =============================================================================
+
+include::../elastic-agent/configuration/outputs/output-logstash.asciidoc[tag=bulk_max_size-setting]
+
+// =============================================================================
+
+include::../elastic-agent/configuration/outputs/output-shared-settings.asciidoc[tag=compression_level-setting]
+
+// =============================================================================
+
+include::../elastic-agent/configuration/outputs/output-shared-settings.asciidoc[tag=escape_html-setting]
+
+// =============================================================================
+
+// tag::index-setting[]
+|
+[id="{type}-index-setting"]
+`index`
+
+| (string) The index root name to write events to.
```

Comment:
For [Filebeat](https://www.elastic.co/guide/en/beats/filebeat/current/logstash-output.html#logstash-index) our docs say:
`The default is the Beat name. For example "filebeat" generates "[filebeat-]8.9.1-YYYY.MM.DD" indices (for example, "filebeat-8.9.1-2017.04.26").`

For Elastic Agent I assume this would be something like `[elastic-agent-]8.9.1-YYYY.MM.DD"`? @joshdover would you happen to know?